### PR TITLE
Feature/tendencies

### PR DESCRIPTION
**Pull request by phawbeck**
Added changes from Jeff (v3.7) and Caroline (3.7.1) for the idealized modifications and outputting tendencies.

Jeff's changes required the merging of module_initialize_les.F into module_initialize_ideal.F (init_les doesn't exist anymore). There was also a large block of code in phys/module_sf_sfclayrev.F that was removed in the latest version (see line 711).

### CODE REVIEW

File: phys/module_sf_sfclayrev.F [hunk 1]
```diff
@@ -708,54 +728,11 @@ SUBROUTINE SFCLAYREV1D(J,UX,VX,T1D,QV1D,P1D,dz8w1d,                &
               Z0T = MAX(Z0T,2.0e-9)
               Z0Q = Z0T
 
-! following paj:
```
Comment (phawbeck) [inline phys/module_sf_sfclayrev.F]: This is the section in question - should this have been removed?

**Comment by ewquon** [comment]
@phawbeck Guessing there has been no update from Jeff to confirm that the results are consistent across WRF versions?

File: phys/module_sf_sfclayrev.F [hunk 2]
```diff
@@ -1043,10 +1016,38 @@ SUBROUTINE SFCLAYREV1D(J,UX,VX,T1D,QV1D,P1D,dz8w1d,                &
 !         ENDIF 
         ELSEIF(XLAND(I)-1.5.LT.0.)THEN                                       
           HFX(I)=FLHC(I)*(THGB(I)-THX(I))                                
-          HFX(I)=AMAX1(HFX(I),-250.)                                       
+          HFX(I)=AMAX1(HFX(I),-250.)
+          
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!JDM 04_2019 Modify Heat Flux Begin
+
+         HFX(I) = 200.0  !W m^-2
```
Comment (ewquon) [inline phys/module_sf_sfclayrev.F]: We need a better solution than hard-coding the heat flux, e.g., adding a namelist option.

**Comment by jsauer-NCAR** [comment]
I would like to merge this pull request. I have spoken with Jeff who is going to immediately reconcile with his own WRF 4.1 modifications that he should were more consistent with his 3.7.1 results.  He will also take a quick look to examine the level of effort required to allow the heat flux to be a run-time parameter.  If it is a heavy lift to do it in a proper-WRF (Registry, config_flags, etc...) type approach then Eliot has a potential work-around solution involving an auxiliary namelist file read in by the module_sf_sfclayrev.F.

**Comment by ewquon** [comment]
The tendencies feature will be merged in #2, and @jmirocha will update the idealized surface layer modeling.

**Pull request closed without merging**